cmd/krew/cmd/root.go

```diff
@@ -98,6 +99,14 @@ func init() {
 	}
 
 	paths = environment.MustGetKrewPaths()
+
+	// Cobra doesn't have a way to specify a two word command (ie. "kubectl krew"), so set a custom usage template
+	// with kubectl in it. Cobra will use this template for the root and all child commands.
+	// See issue #521 for more details on the reason for this: https://github.com/kubernetes-sigs/krew/issues/521
```

Comment:
This last comment line is more apt for the commit message, IMHO. Can you move it there?

Reply:
Oh yeah.  I thought you meant to add it in the code comment, I misunderstood.  Definitely can move it to the commit message instead